pages/general_rules/Procedure.tex

```diff
@@ -118,6 +118,21 @@ \subsection{Start signal}\label{rule:start_signal}
 
 \end{enumerate}
 
+\subsection{Restart Rule}
+\label{rule:restart}
+
+During any test, teams are allowed to perform a single restart per attempt under the following conditions:
+\begin{itemize}
+    \item A restart may only be requested until the robot completes a scoring action.
+    \item A restart incurs a penalty of \SI{30}{\second}, added to the total time.
+    \item The test time will continue running until the team has exited the \Arena{} with the robot.
+    \item After exiting, the team may requeue at the back of the line and begin their attempt again within their remaining time.
+    \item Only one restart is permitted per test attempt.
+\end{itemize}
+
+\textbf{Score Sheet Adjustment:} 
```

Comment:
This does not need to be in the rules.
Just add the checkmark to the scoresheets